skycoin/skywire
# Refactor Summaries

Did you run `make format && make check`?
yes

Fixes #757 

 Changes:	
- Changed `Summary` to `Overview` and `ExtraSummary` to `Summary`.
- Factored out TCPAddr and Online fields into its own struct `NetworkStats`.
- Removed `extraSummaryWithDmsgResp` and updated the new `Summary` accordingly.
- Renamed `getVisorsExtraSummary` to `getAllVisorsSummary`.
- Removed ` TCPAddr` field from `makeSummaryResp`

How to test this PR:
1. Run `make build-ui; make move-built-frontend ; make build; make install`
2. Start Hypervisor and check ui.
